app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
Be more specific in the type annotation in the future indicating a list of what instances you expect to obtain for arguments